src/mlpack/methods/kernel_svm/kernel_svm_main.cpp

```diff
@@ -0,0 +1,431 @@
+/**
+ * @file methods/kernel_svm/kernel_svm_main.cpp
```

Comment:
Can you add this in a separate PR so that it is easy to review? I think we would also have to add the tests for these.